Tests/PreviewTests.stencil

```
@@ -60,9 +60,7 @@ class PreviewTests: XCTestCase {
                                     file: file{% endif %},
                                     testName: testName)
 
-        if let failure,
-           !failure.contains("No reference was found on disk."),
-           !failure.contains("to test against the newly-recorded snapshot") {
+        if let failure {
```

Updated so its clear if something went wrong fetching the submodule/LFS files. We should probably do this on EX too tbf and have CI raise a failure when a snapshot is missing, rather than silently generating it on a temporary runner.